### PR TITLE
Update `The Team` listings

### PR DESCRIPTION
**Pull request by Blooshing**
#indonesian did not have FAMoss listed on the page as all moderators regardless of focus get listed to. This also updates the NAT - will need translators to redo the meta info for their respective language.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by aceticke — CHANGES_REQUESTED** [review]
Staff Log needs updating, I've done this for you here https://github.com/aceticke/osu-wiki/blob/affad5862304ffa6ab61671e455a204cf2e42b15/wiki/People/The_Team/Staff_Log/2022/en.md

**Comment by Walavouchey** [comment]
separate pr was opened for that https://github.com/ppy/osu-wiki/pull/8492

**Comment by Walavouchey** [comment]
hmm with the update to the staff log, the other listings will need to be updated as well to account for the new moves/additions/removals

File: wiki/People/The_Team/Beatmap_Nominators/en.md [hunk 1]
```diff
@@ -209,18 +217,18 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
...
+| ::{ flag=CA }:: [fuhie](https://osu.ppy.sh/users/7620002) |  |
 | ::{ flag=FR }:: [Natsuko](https://osu.ppy.sh/users/8266817) | French |
-| ::{ flag=ZA }:: [Zuid-Afrikaan](https://osu.ppy.sh/users/29282944) | Afrikaans, Dutch |
 
 #### osu!mania
 
 | Name | Additional languages |
 | :-- | :-- |
+| ::{ flag=KR }:: [ArtZone](https://osu.ppy.sh/users/15548034) | Korean |
+| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | German |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/en.md:229]:
```suggestion
| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | German |
| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |  |
```

**Review by aceticke — CHANGES_REQUESTED** [review]
fvrex is a mania bn not standard so this will need to be fixed

File: wiki/People/The_Team/Beatmap_Nominators/es.md [hunk 2]
```diff
@@ -192,6 +195,10 @@ Las tablas a continuación indican el(los) modo(s) de juego en donde cada Beatma
 
 | Nombre | Idiomas adicionales |
 | :-- | :-- |
+| ::{ flag=GB }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
+| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |  |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/es.md:199]:
```suggestion
```

File: wiki/People/The_Team/Beatmap_Nominators/es.md [hunk 3]
```diff
@@ -210,18 +218,18 @@ Las tablas a continuación indican el(los) modo(s) de juego en donde cada Beatma
...
+| ::{ flag=CA }:: [fuhie](https://osu.ppy.sh/users/7620002) |  |
 | ::{ flag=FR }:: [Natsuko](https://osu.ppy.sh/users/8266817) | Francés |
-| ::{ flag=ZA }:: [Zuid-Afrikaan](https://osu.ppy.sh/users/29282944) | Afrikáans, neerlandés |
 
 #### osu!mania
 
 | Nombre | Idiomas adicionales |
 | :-- | :-- |
+| ::{ flag=KR }:: [ArtZone](https://osu.ppy.sh/users/15548034) | Coreano |
+| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | Alemán |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/es.md:230]:
```suggestion
| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | Alemán |
| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |  |
```

File: wiki/People/The_Team/Beatmap_Nominators/fr.md [hunk 4]
```diff
@@ -189,6 +192,10 @@ Les listes ci-dessous indiquent dans quel(s) mode(s) de jeu chaque Beatmap Nomin
 
 | Nom | Langues supplémentaires |
 | :-- | :-- |
+| ::{ flag=GB }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
+| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |  |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/fr.md:196]:
```suggestion
```

File: wiki/People/The_Team/Beatmap_Nominators/pt-br.md [hunk 5]
```diff
@@ -209,18 +217,18 @@ As tabelas listadas abaixo listam os modos de jogo em qual cada Nomeador de Beat
...
+| ::{ flag=CA }:: [fuhie](https://osu.ppy.sh/users/7620002) |  |
 | ::{ flag=FR }:: [Natsuko](https://osu.ppy.sh/users/8266817) | Francês |
-| ::{ flag=ZA }:: [Zuid-Afrikaan](https://osu.ppy.sh/users/29282944) | Africâner, holandês |
 
 #### osu!mania
 
 | Nome | Línguas adicionais |
 | :-- | :-- |
+| ::{ flag=KR }:: [ArtZone](https://osu.ppy.sh/users/15548034) | Coreano |
+| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | Alemão |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/pt-br.md:229]:
```suggestion
| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | Alemão |
| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |  
```

File: wiki/People/The_Team/Beatmap_Nominators/th.md [hunk 6]
```diff
@@ -209,18 +217,18 @@ outdated_since: dc02d48ab01194487cf7bb5633c2982206fa8f53
...
+| ::{ flag=CA }:: [fuhie](https://osu.ppy.sh/users/7620002) |  |
 | ::{ flag=FR }:: [Natsuko](https://osu.ppy.sh/users/8266817) | ฝรั่งเศส |
-| ::{ flag=ZA }:: [Zuid-Afrikaan](https://osu.ppy.sh/users/29282944) | แอฟริกานส์, ดัตช์ |
 
 #### osu!mania
 
 | ชื่อ | ภาษาเพิ่มเติม |
 | :-- | :-- |
+| ::{ flag=KR }:: [ArtZone](https://osu.ppy.sh/users/15548034) | เกาหลี |
+| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | เยอรมัน |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/th.md:229]:
```suggestion
| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | เยอรมัน |
| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |  |
```

File: wiki/People/The_Team/Beatmap_Nominators/tr.md [hunk 7]
```diff
@@ -209,18 +217,18 @@ Aşağıda listelenen tablolar Beatmap Aday Göstericilerin hangi oyun modunda/m
...
+| ::{ flag=CA }:: [fuhie](https://osu.ppy.sh/users/7620002) |  |
 | ::{ flag=FR }:: [Natsuko](https://osu.ppy.sh/users/8266817) | Fransızca |
-| ::{ flag=ZA }:: [Zuid-Afrikaan](https://osu.ppy.sh/users/29282944) | Afrikaanca, Felemenkçe |
 
 #### osu!mania
 
 | İsim | Ek diller |
 | :-- | :-- |
+| ::{ flag=KR }:: [ArtZone](https://osu.ppy.sh/users/15548034) | Korece |
+| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | Almanca |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/tr.md:229]:
```suggestion
| ::{ flag=DE }:: [Feerum](https://osu.ppy.sh/users/4815717) | Almanca |
| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |  |
```

File: wiki/People/The_Team/Beatmap_Nominators/zh.md [hunk 8]
```diff
@@ -191,6 +194,10 @@ BN 分为两类：正式 (Full) 成员和见习 (prob) 成员。
 
 | 名字 | 语言 |
 | :-- | :-- |
+| ::{ flag=GB }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
+| ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |  |
```
Comment (aceticke) [inline wiki/People/The_Team/Beatmap_Nominators/zh.md:198]:
```suggestion
```

**Review by aceticke — CHANGES_REQUESTED** [review]
One last thing to fix

**Review by aceticke — APPROVED** [review]
Looks good

**Review by TicClick — APPROVED** [review]
should be good to go. I don't see significant merit in updating the NAT page translations with English placeholders but I guess that'll do until they are localized

**Pull request merged**